mcp-servers/README.md

```diff
@@ -1,479 +1,386 @@
-    uri: http://mcp-weather:8000/sse
-    port: 8000
-    replicas: 1
-    resources:
-      requests:
-        memory: "256Mi"
-        cpu: "100m"
-      limits:
-        memory: "512Mi"
-        cpu: "200m"
-```
+### Installation Sequence
 
-### Complete Example values.yaml
+**Step 1: Verify Infrastructure Prerequisites**
+```bash
+# Verify OpenShift cluster access
+oc whoami
+oc get nodes
 
-```yaml
-mcp-servers:
-  mcp-weather:
-    deploy: true
-    imageRepository: quay.io/ecosystem-appeng/mcp-weather
-    imageTag: v1.0.0
-    uri: http://mcp-weather:8000/sse
-    port: 8000
-    replicas: 2
-    
-    # Environment variables for weather API
-    env:
-      - name: WEATHER_API_KEY
-        valueFrom:
-          secretKeyRef:
-            name: weather-api-secret
-            key: api-key
-      - name: WEATHER_DEFAULT_LOCATION
-        value: "New York, NY"
-    
-    # Resource limits
-    resources:
-      requests:
-        memory: "512Mi"
-        cpu: "250m"
-      limits:
-        memory: "1Gi"
-        cpu: "500m"
-    
-    # Health check configuration
-    healthCheck:
-      enabled: true
-      path: /health
-      initialDelaySeconds: 30
-      periodSeconds: 10
-    
-    # Service configuration
-    service:
-      type: ClusterIP
-      port: 8000
-      targetPort: 8000
-
-  # Example: Additional MCP server
-  mcp-database:
-    deploy: false
-    imageRepository: quay.io/your-org/mcp-database
-    imageTag: v1.0.0
-    uri: http://mcp-database:8000/sse
-    port: 8000
+# Verify Helm installation
+helm version
 ```
 
-## Usage
+**Step 2: Deploy External Dependencies (Oracle SQLcl MCP only)**
+```bash
+# Deploy Oracle database using the Oracle DB chart
+helm install oracle-db ../oracle-db/helm --namespace <your-namespace>
 
-### Accessing MCP Server Endpoints
+# Wait for Oracle database to be ready
+oc wait --for=condition=ready pod -l app=oracle23ai --timeout=600s
 
-MCP servers expose Server-Sent Events (SSE) endpoints for real-time communication:
+# Verify Oracle secret was created by the database chart
+oc get secret oracle23ai
+```
 
+**Step 3: Install Toolhive Operator**
 ```bash
-# Check MCP server status
-oc get pods -l app.kubernetes.io/name=mcp-servers
+# Add Toolhive Helm repository
+helm repo add toolhive https://stacklok.github.io/toolhive
+helm repo update
 
-# Get service endpoints
-oc get svc -l app.kubernetes.io/name=mcp-servers
+# Install Toolhive CRDs
+helm install toolhive-crds toolhive/toolhive-operator-crds --version 0.0.18
 
-# Port forward for local testing
-oc port-forward svc/mcp-weather 8000:8000
+# Install Toolhive operator with adequate memory resources
+helm install toolhive-operator toolhive/toolhive-operator --version 0.2.6 --namespace <your-namespace>
 
-# Test MCP weather server endpoint
-curl -X GET http://localhost:8000/health
-```
+# IMPORTANT: Increase memory limits to prevent OOMKilled errors
+oc patch deployment toolhive-operator --namespace <your-namespace> --type='json' \
```

Comment:
If you can - better to override this in values.yaml so that it will become easy to manage and automate rather than additional step.